ui/src/file_upload.tsx

```diff
@@ -160,11 +163,12 @@ export const
         const fileArr = Array.from(files)
 
         const errMsg = validateFiles(fileArr)
-        if (errMsg) {
-          errorB(errMsg)
-          return
+        if (errMsg) errorB(errMsg)
```

Comment:
I would preserve the if/return pattern. Also, add braces for both if/else blocks if either block has multiple statements.

Reply:
will do